src/InstagramBasic/Provider.php

```diff
@@ -86,13 +86,15 @@ protected function mapUserToObject(array $user)
      */
     protected function getCodeFields($state = null)
```

Comment:
I guess this whole method should just be:
```php
protected function getCodeFields($state = null): array
{
    $fields = parent::getCodeFields($state);
    $fields['app_id'] = $fields['client_id'];
    unset($fields['client_id']);
    
    return $fields;
}
```

Reply:
Updated